modules/shared/src/main/scala/org/tessellation/schema/TrustValueRefinement.scala

```diff
@@ -0,0 +1,11 @@
+package org.tessellation.schema
+
+import eu.timepit.refined.numeric.Interval
+
+object TrustValueRefinement {
+
+  // Moved this here so that I can exclude this file from scalafmt. This is necessary because of a bug in the current
+  // version of scalafmt that doesn't like the "-" in "-1.0", even though it is valid and the code compiles. scalafmt is
+  // configured to ignore this file.
```

Comment:
imagine you're a newcomer and you see: "moved this here". it doesn't make sense because it's out of context.
anyway, have you tried inline ignoring?

Reply:
I have tried inline ignoring with `// scalafmt:ok` but it didn't work. Ignoring the file was the only thing that worked.